GeosCore/cldj_interface_mod.F90

```diff
@@ -483,6 +483,11 @@ SUBROUTINE Run_CloudJ( Input_Opt, State_Chm, State_Diag, &
        CLDF(1:State_Grid%NZ) = State_Met%CLDF(I,J,1:State_Grid%NZ)
        CLDF(State_Grid%NZ+1) = CLDF(State_Grid%NZ)
 
+       ! Set humidity from input meteorology field
+       ! Convert relative humidity [unitless] from percent to fraction
```

Comment:
Add one more comment: "! Set top of atmosphere relative humidity to 10% of layer below"

Reply:
Thanks for catching that. It's now added in 326c63c.